lib/biran/configurinator.rb

```diff
@@ -103,11 +103,12 @@ def local_config_file_contents
       @local_config_contents = process_config_file(local_config_file)
     end
 
-    def get_secret_contents(app_config)
+    def get_secrets_content(app_config)
       secrets_file_contents = {}
       if File.exist? app_config[:secrets_file_path]
         secrets_file_contents = process_config_file app_config[:secrets_file_path]
       end
+      return app_config[:secrets].deep_merge! secrets_file_contents unless app_config[:secrets].nil?
```

Comment:
Here we early return if we can do the merge, otherwise, just replace what was in app_config[:secrets] with the value of the content from the secrets file.
I thought about seeding the secrets block in the default values with something so it wouldn't be nil, but I couldn't think of anything that was generic and used in 90% of the cases unless we put some useless placeholder.